<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChoixConfig</class>
 <widget class="QDialog" name="ChoixConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>70</y>
     <width>790</width>
     <height>320</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="Default_tab">
    <attribute name="title">
     <string>DEFAULT CONFIGURATIONS</string>
    </attribute>
    <widget class="QPushButton" name="SingleButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>SINGLE</string>
     </property>
    </widget>
    <widget class="QPushButton" name="DGPSButton">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>90</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>DGPS</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RTKButton">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>90</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>RTK STATIC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="RTKButton_2">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>90</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>RTK KINEMATIC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="PPPButton">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>90</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>PPP STATIC</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SBASButton">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>90</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>SBAS</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Customized_tab">
    <attribute name="title">
     <string>CUSTOMIZED CONFIGURATIONS</string>
    </attribute>
    <widget class="QPushButton" name="Custom3Button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>20</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom5Button">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>20</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom6Button">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>20</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom4Button">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>20</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom2Button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>20</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM  2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom9Button">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>160</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom11Button">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>160</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 11</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom12Button">
     <property name="geometry">
      <rect>
       <x>660</x>
       <y>160</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 12</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom10Button">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>160</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 10</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom7Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom8Button">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM 8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Custom1Button">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>120</width>
       <height>110</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
     </property>
     <property name="text">
      <string>CUSTOM  1</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="ReturnButton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>410</y>
     <width>150</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>RETURN</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="WindowTitleEdit">
   <property name="geometry">
    <rect>
     <x>265</x>
     <y>10</y>
     <width>270</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
</string>
   </property>
   <property name="text">
    <string>CONFIGURATION</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="EditeButton">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>350</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius: 15px;
border: 1px solid black;
background-color: rgb(230, 230, 230);</string>
   </property>
   <property name="text">
    <string>CREATE/EDIT CONFIGURATION</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
